model_benefit.go

```diff
@@ -13,27 +13,36 @@ package merge_hris_client
 
 import (
 	"encoding/json"
+	"time"
 )
 
-// Benefit # The Benefit Object ### Description The `Benefit` object is used to represent a Benefit for an employee.  ### Usage Example Fetch from the `LIST Benefits` endpoint and filter by `ID` to show all benefits.
+// Benefit # The Benefit Object ### Description The `Benefit` object is used to represent a benefit that an employee has enrolled in.  ### Usage Example Fetch from the `LIST Benefits` endpoint and filter by `ID` to show all benefits.
 type Benefit struct {
 	Id *string `json:"id,omitempty"`
 	// The third-party API ID of the matching object.
 	RemoteId NullableString `json:"remote_id,omitempty"`
+	// The employee on the plan.
 	Employee NullableString `json:"employee,omitempty"`
 	// The name of the benefit provider.
 	ProviderName NullableString `json:"provider_name,omitempty"`
 	// The type of benefit plan
 	BenefitPlanType NullableString `json:"benefit_plan_type,omitempty"`
 	// The employee's contribution.
-	EmployeeContribution NullableFloat32 `json:"employee_contribution,omitempty"`
+	EmployeeContribution NullableFloat64 `json:"employee_contribution,omitempty"`
```

Comment:
btw this is not backwards compatible but it comes from the API spec so there's not much we can do here other than not changing the API response in the future / using a different field name when we increase number maximums. Either way it's too late now.